### PR TITLE
Implement basic functionality to fetch data from server on html pages

### PR DESCRIPTION
**Pull request by shulijones**
Very simple, just following the walkthrough.

shulijones-step-2020.appspot.com

### CODE REVIEW

**Review by otobin — COMMENTED** [review]
Works on the deployed version!

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -31,6 +31,8 @@ <h2>Shuli's Portfolio</h2>
       <img src="images/ella.jpg" 
           alt="Ella, a small black Havanese, lying on the grass."
           width=300 height=300/>
-    </div>  
+    </div> 
+    <button onclick=getGreeting()>Test out Java servlet</button>
+    <div id=greeting></div>
```
Comment (kyracq) [inline portfolio/src/main/webapp/index.html:36]: Missing quotations around greeting and getGreeting()

**Comment by shulijones** [comment]
> I get a 404 error when I press the button on the index homepage. I'm previewing the file from your dev branch. I'm still working on the first two tutorials so I'm not exactly sure where your error.

@otobin , could you try again from go/shulijones-app ? I'm not sure why the preview isn't working but I just deployed and hopefully that will work.

**Review by kyracq — COMMENTED** [review]
~~I get a "400: Invalid request" error, but it might just be an issue with HTMLPreview if it works in your dev server~~

Works at go/shulijones-app!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (shulijones) [reply portfolio/src/main/webapp/index.html:36]: Thanks!

**Review by nsri-google — APPROVED** [review]

**Review by anicr7 — APPROVED** [review]

**Pull request merged**